packages/commons/package.json

```diff
@@ -26,13 +26,17 @@
     "@zodios/express": "10.6.1",
     "axios": "1.7.1",
     "connection-string": "4.4.0",
+    "date-fns": "3.6.0",
+    "date-fns-tz": "^3.1.3",
+    "handlebars": "^4.7.8",
     "jsonwebtoken": "9.0.2",
     "jwks-rsa": "3.1.0",
     "kafkajs": "2.2.4",
     "mongodb": "5.9.2",
     "pagopa-interop-models": "workspace:*",
     "pg-promise": "11.6.0",
     "ts-pattern": "5.1.1",
+    "puppeteer": "^22.8.0",
```

Comment:
Same here

Reply:
Done with commit [169abfc](https://github.com/pagopa/interop-be-monorepo/pull/520/commits/169abfcea2cfdd1497fece6b5eb7e256c582f3df)

Reply:
Done with commit [169abfc](https://github.com/pagopa/interop-be-monorepo/pull/520/commits/169abfcea2cfdd1497fece6b5eb7e256c582f3df)